managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -463,6 +474,10 @@ public Result create(UUID customerUUID) {
       LOG.info("Create for {}.", customerUUID);
       // Get the user submitted form data.
       formData = (ObjectNode) request().body().asJson();
+      if (validateEncryption(formData))
+      {
+        return ApiResponse.error(BAD_REQUEST, "Node to node encryption must required to enable for client to node encryption.");
```

Comment:
Please rephrase. It is not completely clear what it wants to say.

Reply:
Done